packages/cli/src/util/deploy/parse-target.ts

```diff
@@ -37,5 +37,25 @@ export default function parseTarget(
     return 'production';
   }
 
+  const VERCEL_ENV = process.env.VERCEL_ENV;
+  if (VERCEL_ENV) {
+    if (!['preview', 'staging', 'production'].includes(VERCEL_ENV)) {
```

Comment:
I don't think we should allow `staging` here. 

Also cc @lucleray who knows more about this target.